packaging/windows/WindowsLauncher.cs.in

```diff
@@ -0,0 +1,226 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Media;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Windows.Forms;
+
+namespace OpenRA
+{
+	class WindowsLauncher
+	{
+		[DllImport("user32.dll")]
+		static extern int SendMessage(IntPtr hwnd, uint message, uint wParam, IntPtr lParam);
+
+		[DllImport("shell32.dll")]
+		static extern IntPtr SHGetFileInfo(string pszPath, uint dwFileAttributes, ref SHFILEINFO psfi, uint cbSizeFileInfo, uint uFlags);
+
+		[DllImport("user32.dll")]
+		public extern static bool DestroyIcon(IntPtr handle);
+
+		struct SHFILEINFO
+		{
+			public IntPtr hIcon; // original type: HICON
```

Comment:
Comments usually go on their own line, starting with a capital letter. (Same for the two other comments below.)